dist/redhat/scylla.spec

```diff
@@ -54,7 +54,7 @@ Group:          Applications/Databases
 Summary:        The Scylla database server
 License:        AGPLv3
 URL:            http://www.scylladb.com/
-Requires:       %{product}-conf  = %{version} %{product}-python3 = %{version}
+Requires:       %{product}-conf  = %{version}-%{release} %{product}-python3 = %{version}
```

Comment:
What about scylla-python3 here?

Could have a %{verrel} macro to reduce noise.

Reply:
ahh, i missed that. i'd prefer to keep using `%{version}-%{release}`. as it's more idiomatic. see https://src.fedoraproject.org/rpms/kernel/blob/rawhide/f/kernel.spec . i cannot think of another rpm package receipe which insists a stronger dependency on exactly the same version of its dependencies.